src/go/rpk/pkg/cli/cmd/redpanda/start.go

```diff
@@ -162,22 +163,24 @@ func NewStartCommand(fs afero.Fs, launcher rp.Launcher) *cobra.Command {
 			// Thus, we can ignore any env / flags that would come from rpk
 			// configuration itself.
 			cfg = cfg.FileOrDefaults()
-			clusterProperties := make(map[string]interface{})
-			if sandbox {
-				fmt.Fprintln(os.Stderr, "WARNING: This is a setup for development purposes only; in sandbox mode your clusters may run unrealistically fast and data can be corrupted any time your computer shuts down uncleanly.")
-				cfg.Redpanda.DeveloperMode = true
-				setDevModeFlags(cmd)
-				clusterProperties["topic_partitions_per_shard"] = 1000
-				clusterProperties["group_topic_partitions"] = 1
-				clusterProperties["auto_create_topics_enabled"] = true
+			if cfg.Redpanda.DeveloperMode && len(sandbox) == 0 {
+				sandbox = "default"
 			}
-			if cfg.Redpanda.DeveloperMode {
-				clusterProperties["storage_min_free_bytes"] = 0
-			}
-			if len(clusterProperties) > 0 {
-				err := setClusterProperties(fs, cfg.FileLocation(), clusterProperties)
-				if err != nil {
-					fmt.Fprintf(os.Stderr, "unable to set cluster properties: %v\n", err)
+			if len(sandbox) > 0 {
```

Comment:
Unnecessary conditional, the switch below can have a case `""`

Reply:
continue on `""` case then?

Reply:
yep, basically catch `""` and then do nothing, which continues to the logic below